src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditorWidget.java

```diff
@@ -1297,7 +1310,10 @@ else if (item.getType() == "spelling")
             clazz = lintStyles_.spelling();
          
          int id = editor_.getSession().addMarker(range, clazz, "text", true);
-         annotations_.add(new AnchoredAceAnnotation(item.asAceAnnotation(), range, id));
+         if (item.getType() == "spelling")
```

Comment:
Nit: per [ancient agreement](https://github.com/rstudio/rstudio/pull/2002), we should be using StringUtil.equals for string comparisons, not `==`. In practice this doesn't matter a whole lot (and other places in this file and no doubt many others don't do this either).